src/main/java/uk/gov/hmcts/divorce/caseworker/service/task/GenerateNoticeOfProceeding.java

```diff
@@ -68,6 +73,12 @@ private void generateSoleNoticeOfProceedings(CaseData caseData, Long caseId) {
         final boolean isApplicant1Represented = caseData.getApplicant1().isRepresented();
         final boolean isApplicant2Represented = caseData.getApplicant2().isRepresented();
 
+        if (isApplicant1Represented
+            && caseData.getApplicant1().getSolicitor().hasOrgId()
```

Comment:
is this applicable for paper cases ?

Reply:
this is for online only sole and joint application as confirmed with Lalita